src/Symfony/Component/Validator/Tests/ValidatorBuilderTest.php

```diff
@@ -42,7 +42,7 @@ public function testAddObjectInitializer()
 
     public function testAddObjectInitializers()
     {
-        $this->assertSame($this->builder, $this->builder->addObjectInitializers(array()));
+        $this->assertSame($this->builder, $this->builder->addObjectInitializers([]));
```

Comment:
it looks like your base your branch on an old version of the 4.2 branch - can you please rebase on latest 4.2 to get rid of these?

Reply:
Will do